go/pkg/bertyprotocol/api_contact.go

```diff
@@ -7,9 +7,13 @@ import (
 
 	"berty.tech/berty/v2/go/pkg/errcode"
 	"berty.tech/berty/v2/go/pkg/protocoltypes"
+	"berty.tech/berty/v2/go/pkg/tyber"
 )
 
-func (s *service) ContactAliasKeySend(ctx context.Context, req *protocoltypes.ContactAliasKeySend_Request) (*protocoltypes.ContactAliasKeySend_Reply, error) {
+func (s *service) ContactAliasKeySend(ctx context.Context, req *protocoltypes.ContactAliasKeySend_Request) (_ *protocoltypes.ContactAliasKeySend_Reply, err error) {
+	ctx, _, endSection := tyber.Section(ctx, s.logger, "Sending contact alias key")
```

Comment:
Maybe we can set this kind of section inside a gRPC middleware ?

Reply:
We could but we don't want it on all the calls so maybe not the best way